serde-helpers/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "serde-helpers"
```

Comment:
is this a sufficiently common pattern that we want to add yet another helper crate?

Reply:
Moved into `omicron-common` in 52bd108a4